include/gc_inline.h

```diff
@@ -31,6 +31,10 @@
 #include "gc.h"
 #include "gc_tiny_fl.h"
 
+# ifdef _MSC_VER
```

Comment:
In case of public headers (like gc_inline.h), it would be better to include additional headers only for the condition which require it, e.g. in this case it could be: !defined(GC_PREFETCH_FOR_WRITE) && !defined(GC_NO_PREFETCH_FOR_WRITE) && defined(_MSC_VER) && (defined(_M_X64) || ...)